src/ui/server/helpers/get-field-by-id/index.ts

```diff
@@ -3,19 +3,22 @@ import { objectHasProperty } from '../object';
 /**
  * getFieldById
  * Get a field from an object by the field ID
- * @param {Object} Fields
- * @param {String} Field ID
+ * if field has optional prefix, then add to the id
+ * @param {Object} fields
+ * @param {String} fieldId ID
+ * @param {String} fieldIdPrefix optional field if prefix
  * @returns {Object} Field or an object with the provided field ID.
  */
-const getFieldById = (fields: object, fieldId: string) => {
+const getFieldById = (fields: object, fieldId: string, fieldIdPrefix?: string) => {
   if (objectHasProperty(fields, fieldId)) {
     return {
-      id: fieldId,
+      // if prefix, then add prefix before fieldId, else set id as fieldId
+      id: fieldIdPrefix ? `${fieldIdPrefix}-${fieldId}` : fieldId,
       ...fields[fieldId],
     };
   }
 
-  return { id: fieldId };
+  return { id: fieldIdPrefix ? `${fieldIdPrefix}-${fieldId}` : fieldId };
```

Comment:
Duplicated lines here, could have `id` constant with this condition and then simplifies line 16 and 21